plugin-server/src/logs-ingestion/logs-ingestion-consumer.ts

```diff
@@ -162,6 +198,50 @@ export class LogsIngestionConsumer {
         )
     }
 
+    private async emitUsageMetrics(usageStats: UsageStatsByTeam): Promise<void> {
+        if (usageStats.size === 0) {
+            return
+        }
+
+        const timestamp = castTimestampOrNow(null, TimestampFormat.ClickHouse)
+
+        const metricsPromises: Promise<void>[] = []
+        for (const [teamId, stats] of usageStats) {
+            metricsPromises.push(
+                this.produceUsageMetric(teamId, 'bytes_received', stats.bytesReceived, timestamp),
+                this.produceUsageMetric(teamId, 'records_received', stats.recordsReceived, timestamp),
+                this.produceUsageMetric(teamId, 'bytes_ingested', stats.bytesAllowed, timestamp),
+                this.produceUsageMetric(teamId, 'records_ingested', stats.recordsAllowed, timestamp),
+                this.produceUsageMetric(teamId, 'bytes_dropped', stats.bytesDropped, timestamp),
+                this.produceUsageMetric(teamId, 'records_dropped', stats.recordsDropped, timestamp)
+            )
+        }
+
+        await Promise.all(metricsPromises)
+    }
+
+    private produceUsageMetric(teamId: number, metricName: string, count: number, timestamp: string): Promise<void> {
+        if (count === 0) {
+            return Promise.resolve()
+        }
+        return this.kafkaProducer!.produce({
```

Comment:
Unless I'm missing something this will use the configured producer which currently points at Warpstream.

This needs to point at the main MSK one, so we need to configure two producers and use the correct ones for the different targets.